.github/workflows/rdbms-integration-test.yml

```diff
@@ -0,0 +1,72 @@
+name: RDBMS Integration Tests
+on:
+  push:
+    branches:
+      - 'main'
+      - 'stable/**'
+      - 'release/**'
+  pull_request:
```

Comment:
🔧 Workflow could also run if something changes in the `search-domain` module as the filters are directly used in the mapper files.

Reply:
Good idea, extended the pattern